website/src/queries-logic.js

```diff
@@ -164,41 +176,26 @@ function send(requestPromise, successFn, failureFn) {
       () => decrementCounters());
 }
 
-/** Fills the first part of a row. */
-function build_fork_element_html(table_body, combined_name, num_stars, num_forks) {
-  const NEW_ROW = $('<tr>', {id: extract_username_from_fork(combined_name), class: "useful_forks_repo"});
-  table_body.append(
-      NEW_ROW.append(
-          $('<td>').html(getRepoCol(combined_name, false)).attr("value", combined_name),
-          $('<td>').html(UF_TABLE_SEPARATOR + getStarCol(num_stars)).attr("value", num_stars),
-          $('<td>').html(UF_TABLE_SEPARATOR + getForkCol(num_forks)).attr("value", num_forks)
-      )
-  );
-  return NEW_ROW;
-}
-
-/** Add bold to the date text if the date is earlier than the queried repo. */
-function compareDates(date, html) {
-  return REPO_DATE <= new Date(date) ? `<strong>${html}</strong>` : html;
-}
```

Comment:
Would you mind bringing this piece of code back above that method so that Git doesn't think it was touched by this PR?